CHANGES.md

```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## 1.6.7 (core) / 0.22.7 (libraries)
+
+### New
+- Added a new `run_retries.retry_on_op_or_asset_failrues` setting that can be set to false to make run retries only occur when there is an unexpected failure that crashes the run, allowing run-level retries to co-exist more naturally with op or asset retries. See [the docs](https://docs.dagster.io/deployment/run-retries#combining-op-and-run-retries) for more information.
+- `dagster dev` now sets the environment variable `DAGSTER_IS_DEV_CLI` allowing subprocesses to know that they were launched in a development context.
+- [ui] The Asset Checks page was updated to show more information on the page itself rather than in a dialog.
+
+### Bugfixes
+- [ui] Fixed an issue the UI disallowed creating a dynamic partition if its name contained the “|” pipe character.
+
+### Experimental
+- The new `@multi_observable_source_asset` decorator enables defining a set of assets that can be observed together with the same function.
+- [dagster-embedded-elt] New Asset Decorator `@sling_assets` and Resource `SlingConnectionResource` have been added for the `[dagster-embedded-elt.sling](http://dagster-embedded-elt.sling)` package. Deprecated `build_sling_asset`, `SlingSourceConnection` and `SlingTargetConnection`.
```

Comment:
Can we add a section that I forgot to add?

```
- Added support for op-concurrency aware run dequeuing for the `QueuedRunCoordinator`
```

Reply:
done!